src/Provider.js

```diff
@@ -21,7 +21,7 @@ export class Provider extends Component {
         return createElement(
```

Comment:
I also want to use JSX here, but I will do it in a separate PR.

Reply:
And you can rewrite it to FC when you are it :D

Reply:
Thanks for the idea :) I'll look into it.